epub34/rs/index.html

```diff
@@ -1205,22 +1205,22 @@ <h3>Cascading Style Sheets (CSS)</h3>
 							[=EPUB content documents=].</p>
 					</li>
 					<li>
-						<p id="confreq-css-overrides">SHOULD NOT override the EPUB creator's style sheets, but SHOULD do
-							so in a way that preserves the Cascade when necessary: through a user agent style sheet, or
-							[[html]] [^html-global/style^] attributes.</p>
+						<p id="confreq-css-overrides">MAY override parts of the creator's style sheets (for, e.g., ergonomic or 
+							user interface reasons). If it does, it SHOULD preserve as much as possible the cascade model of CSS.
+							Reading system developers are expected to also publicly document the user agent style sheets and how they
+							interact with EPUB creator's style settings.</p>
 					</li>
 					<li>
-						<p id="confreq-css-user-styles">It MAY override parts of the EPUB creator's style sheet because
-							of user interaction.</p>
+						<p id="confreq-css-user-styles">
+							It MAY provide the user with an affordance that reverts the display to the author's stylesheet.</p>
```

Comment:
Similar to the comment I made about the authoring spec, I don't see how these two new bullets fit with the rest of the list. Here we're talking about what reading systems have to do to support css and then veer off into how to override the publication's css.

I'd take these bullets and make a new subsection on overriding the publication's css.

I'd try to make some changes, but the more I look at the sentence about preserving the cascade the less I understand what it's trying to achieve. I also have reservations about getting into reading system affordances.

Reply:
Setting styling baking into the EPUB: creators. Users/readers may have the ability to virtually change styling via the RS' user interface.

Reply:
> But what does it mean to preserve as much as possible the cascade model of CSS?

I have taken it to mean that styling applied by the RS, either at the behest of the user or for other reasons, should be done using the cascade. But that is more "using the cascade model", not "preserving". Trying to wrap my mind around it now, and it is really unclear - how would a RS degrade the cascade model? Maybe if they implemented a custom CSS handler that calculated specificity in a non-standard way?

>  It sounds like it's suggesting that reading systems are defining their own styling systems outside of CSS.

Some do of course, and I suppose they could break the cascade.